CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+## 2.15.0 - 2017-09-12
+- The public_url column has to be unique by default. This because it was not supported by the 
+  application but the db schema allowed double entries with the same pulbic_url.
+  
+  There is now validation and a change in the column property. Be careful with schema updates 
+  because it can lead to failures when there allready double entries for the public_url.
+  
+  This all can be disable if find any problems with upgrading by setting the strict property 
+  to false in the config:  
+  
+  ```
+    zicht_url:
+        strict: false
```

Comment:
Ik zou dit anders noemen, `strict` kan heel veel verschillende dingen bevatten terwijl het nu enkel `unique_public_url` doet, ik denk dat daarom een dergelijke (niet persee die) naam beter is, dan weet je direct wat het doet als je het in de config ziet staan.

Dit geld dan ook voor alle andere voorkomingen van het woord strict, i.e. de `StrictListener` -> `UniquePublicUrlListener`.